### ISSUE TITLE
Animated emotes start from zero

### ISSUE THREAD

**Issue by Empakt** [crackededed/Xtra#185]
Unlike the bttv and 7tv on desktop where the playback of animated emotes is synched between messages, in the app right now every new message starts the playback from zero.
Emotes like Aware (https://betterttv.com/emotes/6151c623b63cc97ee6d39040) and most notably Countdown (https://betterttv.com/emotes/60cd19468ed8b373e4216071) basically never have time to play as the playback starts from scratch with each message.
Solution would be to have the new message's emotes just continue with the previous ones, just like on desktop.
This is maybe a nitpick issue, but it definitely worsens the chat experience.

**Comment by imBigo** [comment]
Yeah so basically any animated emotes aren't in sync. I have also noticed this idk if there's an easy fix for this? @crackededed

**Comment by Empakt** [comment]
> Yeah so basically any animated emotes aren't in sync. I have also noticed this idk if there's an easy fix for this? @crackededed

**Comment by Empakt** [comment]
I wouldn't say any aren't in sync, emotes in the same message are synced, but that's about it

**Comment by crackededed** [comment]
idk maybe adding another image library will fix this
